docs/data-model/zst.md

```diff
@@ -408,32 +404,29 @@ gracie
 0
 ===
 ```
-
 To see the detailed ZST structure described as ZSON, you can use the `zst`
 command like this:
 ```
-zst inspect -Z -trailer hello.zst
+zed dev dig section -Z 1 hello.zst
```

Comment:
Maybe use mdtest? Or maybe not since then you'd lose the ability to include inline comments.

Reply:
We can improve later